src/ixdat/techniques/ms.py

```diff
@@ -92,10 +93,16 @@ def grab(
                 Defaults to False, but in grab_flux it defaults to True.
             include_endpoints (bool): Whether to ensure tspan[0] and tspan[-1] are in t
         """
+        if plugins.use_si_quant and item.startswith("n_dot_"):
+            return self.grab_flux(
+                item.lstrip("n_dot_"),
```

Comment:
`strip` in Python **always** removes as many of the characters in the string you give as argument to `strip` it can find, **not** the as a substring. *Technically* I think you are fine, since no molecules start with any of the characters `["n", "d", "o", "t", "_"]`, but it is a bug since it doesn't work as intended and only works by accident, and should therefore be changed to `removeprefix`.

Reply:
thanks, I made a false assumption about stripping!. Changed to `removeprefix`.